image.yaml

```diff
@@ -18,6 +18,7 @@ envs:
       value: "/opt/interconnect"
```

Comment:
When pulling the base image.  Should use specific tag:
"from: "jboss/base-rhel7:1.2" instead of latest
(not directly related to this commit).